src/Quasar/Data/Json.purs

```diff
@@ -0,0 +1,65 @@
+{-
+Copyright 2017 SlamData, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-}
+
+module Quasar.Data.Json where
+
+import Prelude
+
+import Data.Generic.Rep (class Generic)
+import Data.MediaType (MediaType(..))
+import Data.Newtype (class Newtype)
+import Quasar.Data.MediaTypes (applicationJSON, applicationLDJSON)
+
+type OptionsR =
+  { encoding ∷ EncodingStyle
+  , precision ∷ PrecisionMode
+  }
+
+newtype Options = Options OptionsR
+
+derive instance eqOptions ∷ Eq Options
+derive instance ordOptions ∷ Ord Options
+derive instance newtypeOptions ∷ Newtype Options _
+derive instance genericOptions ∷ Generic Options _
+
+data PrecisionMode = Readable | Precise
+
+derive instance eqPrecisionMode ∷ Eq PrecisionMode
+derive instance ordPrecisionMode ∷ Ord PrecisionMode
+derive instance genericPrecisionMode ∷ Generic PrecisionMode _
```

Comment:
Yeah, for a codec in SlamData... or for that in theory at least, I can't remember whether I switched it or not. If we don't have it here it can't be defined anywhere anyway :wink: